scripts/rename_session_windows.py

```diff
@@ -344,7 +414,15 @@ def print_programs(server: Server, options: Options):
 
     for pane in panes_programs:
         if pane.program:
-            print(f'{pane.program} -> {substitute_name(pane.program, options.substitute_sets)}')
+            program_name = substitute_name(pane.program, options.substitute_sets)
+            if options.icon_style in ['icon', 'name+icon']:
```

Comment:
Make a common function for this. Its being duplicated in rename_window()

Reply:
Extracted a common function